plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -364,6 +364,15 @@ private function get_feature_definitions() {
 					'is_legacy'          => true,
 					'is_experimental'    => false,
 				),
+				'email_block_editor'     => array(
+					'name'               => __( 'Email Block Editor (beta)', 'woocommerce' ),
```

Comment:
Can we use `block_email_editor` instead? That is the name we have been using for the project.
This part should also be  Alpha, not Beta yet.